openlane_update_tool/action.yml

```diff
@@ -0,0 +1,37 @@
+name: Update OpenLane Tool
+
+inputs:
+  tool_name:
+    description: Name of the tool to update.
+    required: true
+  pdk_root:
+    description: Path to the PDK folder.
+    default: .
+  build_ol:
+    description: Build docker image after tool update.
+    default: true
+
+runs:
+  using: composite
+  steps:
+
+    - name: Update tool hash
+      shell: bash
+      run: python3 .github/scripts/update_tools.py ${{ inputs.tool_name }}
```

Comment:
isn't the `update_tools.py` script tied to a particular repo?

Reply:
Yes, for OpenLane. This action is intended to be used for workflows that use that repo.

Reply:
if the action is tied to openlane, shouldn't it live in https://github.com/The-OpenROAD-Project/OpenLane/tree/master/.github/workflows ?

Reply:
The goal of these actions is to allow the orchestration of the test and update OL version without touching OL repo. We might consider moving the action later if OL devs agree.

Reply:
oh I see, so this is the local equivalent of https://github.com/The-OpenROAD-Project/OpenLane/blob/master/.github/workflows/tool_updater.yml, to specify that you want to update one of the tool to the latest version before running `openlane_run`?